week-5/C-promises/exercise.js

```diff
@@ -44,7 +53,14 @@ function exercise3() {
   of your promise and insert it into the #exercise4 element
 */
 function exercise4() {
-  var promise4 = concatPromise()
+  var promise4 = concatPromise();
+  promise4
+    .then(result => {
+      return ` ${result} Francisca `;
+    })
+    .then(value => {
+      document.body.querySelector("#exercise4").innerHTML = value;
```

Comment:
This is nice & clear what you're doing, I like it 👍 

There is another way of doing it (which isn't necessarily better, just letting you know), where you concatenate the string and update the DOM in 1 step:

```js
.then((result) => {
  document.querySelector('#exercise4').textContent = `${result} Ali`;
})
```